proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -1284,6 +1291,13 @@ message ColumnStatisticsRequest {
   optional int32 unique_value_limit = 4;
 }
 
+message SliceRequest {
+  Ticket result_id = 1;
+  TableReference source_id = 2;
+  sint64 start_row = 3 [jstype=JS_STRING];
+  sint64 end_row = 4 [jstype=JS_STRING];
```

Comment:
Similarly, I wonder if these should be named `first_position_inclusive` / `last_position_exclusive` to better match the java names, and have a comment at this level.